<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>266</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>369</width>
    <height>222</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Instant AI-Configuration</string>
  </property>
  <widget class="QComboBox" name="cmbDevice">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblDevice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>90</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Instant AI settings</string>
   </property>
   <widget class="QComboBox" name="cmbChannelStart">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="lblValueRange">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>84</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Value range:</string>
    </property>
    <property name="buddy">
     <cstring>cmbValueRange</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelCount">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>52</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel count:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannelStart">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>22</height>
     </size>
    </property>
    <property name="text">
     <string>Channel start:</string>
    </property>
    <property name="buddy">
     <cstring>cmbChannelStart</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbValueRange">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>84</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbChannelCount">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>52</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblDevice_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Profile:</string>
   </property>
   <property name="buddy">
    <cstring>cmbDevice</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProfilePath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbDevice</tabstop>
  <tabstop>cmbChannelStart</tabstop>
  <tabstop>cmbChannelCount</tabstop>
  <tabstop>cmbValueRange</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
